src/components/MedicationGroup.js

```diff
@@ -52,6 +58,19 @@ export default function MedicationGroup({
             <img src={editIcon} alt="bearbeiten" height="16px" />
           </IconButton>
         </>
+      ) : (
+        <IconButton
+          right="10px"
+          bottom="10px"
+          onClick={() => handleCheckClick(id)}
+          position="absolute"
+        >
+          {check ? (
+            <img src={checkIcon} alt="löschen" height="20px" />
+          ) : (
+            <img src={checkIconDisabled} alt="löschen" height="20px" />
```

Comment:
I think here the `alt` texts could be described a bit better? It's not both for deleting?

Reply:
actually I just forgot to change it